components/dashboard/src/settings/PersonalAccessTokens.tsx

```diff
@@ -20,10 +20,7 @@ function PersonalAccessTokens() {
 
     return (
         <div>
-            <PageWithSettingsSubMenu
-                title="Preferences"
-                subtitle="Manage your Personal Access Tokens to access the Gitpod API."
-            >
+            <PageWithSettingsSubMenu title="Preferences" subtitle="Manage your personal access tokens.">
```

Comment:
**suggestion**: Minor title rename to match the sidebar menu option.

```suggestion
            <PageWithSettingsSubMenu title="Access Tokens" subtitle="Manage fine-grained personal access tokens.">
```

Alternatively, this could also change the subtitle:

```suggestion
            <PageWithSettingsSubMenu title="Access Tokens" subtitle="Manage your personal access tokens.">
```